packages/app/app/components/ArtistView/ArtistLink/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Button } from 'semantic-ui-react';
+import FontAwesome from 'react-fontawesome';
+
+const shell = require('electron').shell;
+
+const ArtistLink = ({ artistLink, size, label='Visit', circular=false}) => (
+  <Button
+    primary
+    onClick={() => shell.openExternal(artistLink)}
+    size={size}
+    circular={circular}
+    aria-label={'Visit Artist Profile'}
+  >
+    <FontAwesome name='link' /> {label}
```

Comment:
This label should be localized, can you use the localization API and create a string in en.json?